remappers/hostmetrics/cpu.go

```diff
@@ -61,23 +85,8 @@ func addCPUMetrics(metrics pmetric.MetricSlice, resource pcommon.Resource, datas
 		}
 	}
 
-	totalNorm := totalPercent / float64(numCores)
-	idleNorm := idlePercent / float64(numCores)
-	systemNorm := systemPercent / float64(numCores)
-	userNorm := userPercent / float64(numCores)
-	stealNorm := stealPercent / float64(numCores)
-	iowaitNorm := iowaitPercent / float64(numCores)
-	niceNorm := nicePercent / float64(numCores)
-	irqNorm := irqPercent / float64(numCores)
-	softirqNorm := softirqPercent / float64(numCores)
-
-	addMetrics(metrics, resource, dataset,
-		metric{
-			dataType:  pmetric.MetricTypeSum,
-			name:      "system.cpu.cores",
-			timestamp: timestamp,
-			intValue:  &numCores,
-		},
+	// Add all metrics that are independent of cpu logical count.
+	addMetrics(out, dataset, emptyMutator,
```

Comment:
We are putting out the mtrics in different metric slices here?

Reply:
Yes, you can choose to use the same src by passing in the `MetricSlice` created from the `ScopeMetric`